src/CMakeLists.txt

```diff
@@ -30,3 +31,9 @@ if(CLANG_FORMAT)
   add_custom_target(CL_FORMAT_CSRCS COMMAND ${CLANG_FORMAT_EXEC} -i -style=file ${ALL_CSRCS})
   add_dependencies(torch_xpu_ops CL_FORMAT_CSRCS)
 endif()
+
+if(USE_ONEMKL)
+  target_compile_options(torch_xpu_ops PRIVATE "-DUSE_ONEMKL")
+  target_include_directories(torch_xpu_ops PUBLIC ${TORCH_XPU_OPS_ONEMKL_INCLUDE_DIR})
+  target_link_libraries(torch_xpu_ops PUBLIC ${TORCH_XPU_OPS_ONEMKL_LIBRARIES})
+endif()
```

Comment:
Move lines to `BuildOnWindows.cmake` and `BuildOnLinux.cmake`. They are a part of CMAKE target `torch_xpu_ops`. The file `src/CMakeLists.txt` is not CMAKE target related.

Reply:
Changes done.